python/tank/platform/engine.py

```diff
@@ -176,6 +176,7 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt.QtCore = base_def.get("qt_core")
         qt.QtGui = base_def.get("qt_gui")
         qt.TankDialogBase = base_def.get("dialog_base")
+        qt.shiboken = base_def.get("shiboken")
```

Comment:
We should update these [unit tests](https://github.com/shotgunsoftware/tk-core/blob/master/tests/util_tests/test_qt_importer.py) to validate the new `shiboken` module